samples/automl/automlTranslationModel.js

```diff
@@ -0,0 +1,481 @@
+  const myModel = {
+    displayName: modelName,
+    datasetId: datasetId,
+    translationModelMetadata: {},
+  };
+
+  // Create a model with the model metadata in the region.
+  client
+    .createModel({parent: projectLocation, model: myModel})
+    .then(responses => {
+      const operation = responses[0];
+      const initialApiResponse = responses[1];
+
+      console.log('Training operation name: ', initialApiResponse.name);
+      console.log('Training started...');
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      const model = responses[0];
+      console.log(model);
+      // Retrieve deployment state.
+      var deploymentState = '';
+      if (model.deploymentState === 1) {
+        deploymentState = 'deployed';
+      } else if (model.deploymentState === 2) {
+        deploymentState = 'undeployed';
+      }
+
+      // Display the model information.
+      console.log('Model name: ', model.name);
+      console.log('Model id: ', model.name.split('/').pop(-1));
+      console.log('Model display name: ', model.displayName);
+      console.log('Model create time:');
+      console.log('\tseconds: ', model.createTime.seconds);
+      console.log('\tnanos: ', model.createTime.nanos);
+      console.log('Model deployment state: ', deploymentState);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_translation_createModel]
+}
+
+function listModels(projectId, computeRegion, filter_) {
+  // [START automl_translation_listModels]
+  const automl = require('@google-cloud/automl');
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const filter_ = `filter expressions, must specify field, e.g. "translationDatasetMetadata:*”`;
+
+  // A resource that represents Google Cloud Platform location.
+  const projectLocation = client.locationPath(projectId, computeRegion);
+
+  // List all the models available in the region by applying filter.
+  client
+    .listModels({parent: projectLocation, filter: filter_})
+    .then(responses => {
+      const model = responses[0];
+
+      // Display the model information.
+      console.log('List of models:');
+      for (let i = 0; i < model.length; i += 1) {
+        console.log('Model name: ', model[i].name);
+        console.log('Model id: ', model[i].name.split('/').pop(-1));
+        console.log('Model display name: ', model[i].displayName);
+        console.log('Model dataset id: ', model[i].datasetId);
+        if (model[i].modelMetadata === 'translationModelMetadata') {
+          console.log('Translation model metadata:');
+          console.log(
+            '\tBase model: ',
+            model[i].translationModelMetadata.baseModel

```

Comment:
Move things like into a shorter variable name, to avoid multi-line `console.log` statements?

Apply this comment throughout your PR.